tests/test_execute/test_complex_outputs_unrun_auto.ipynb

```diff
@@ -430,7 +430,7 @@
      "execution_count": 5,
      "metadata": {
       "filenames": {
-       "image/png": "/private/var/folders/t2/xbl15_3n4tsb1vr_ccmmtmbr0000gn/T/pytest-of-chrisjsewell/pytest-679/test_complex_outputs_unrun_aut0/source/_build/jupyter_execute/complex_outputs_unrun_22_0.png"
+       "image/png": "/private/var/folders/_w/bsp9j6414gs4gdlnhhcnqm9c0000gn/T/pytest-of-matthewmckay/pytest-37/test_complex_outputs_unrun_aut0/source/_build/jupyter_execute/complex_outputs_unrun_22_0.png"
```

Comment:
@choldgraf I just updated a whole bunch of metadata in the tests in `upgrade to sphinx 4` and the tests actually ignore the metadata

https://github.com/executablebooks/MyST-NB/blob/b132a50a8fdceaa86950a23c854cf537d0a1d2bc/tests/conftest.py#L23

I don't see an easy way to generate these fixtures with generic values.

Reply:
as mentioned, the actual tests ignore these values when diffing.
if you wanted to reduce the diff though, it would be possible to add a pre-commit hook, which replaces all the "non-deterministic" content with deterministic ones